deps/rabbitmq_shovel/test/amqp10_dynamic_SUITE.erl

```diff
@@ -168,6 +170,169 @@ simple_amqp10_src(Config) ->
               ok
       end).
 
+message_prop_conversion(Config) ->
```

Comment:
These are the same tests as on 3.12.x. With minimal changes, they produce the same output.